source/introduction/kafka-connect.txt

```diff
@@ -4,6 +4,9 @@
 Kafka and Kafka Connect
 =======================
 
+.. meta::
```

Comment:
question: should we describe MongoDB as a datastore? I think perhaps database is more appropriate.

Reply:
Up to you - looks like we use "datastores such as MongoDB" in the page content which is likely where the AI got this phrasing